plugins/src/main/java/org/smartregister/fhir/gateway/plugins/PractitionerDetailsEndpointHelper.java

```diff
@@ -88,7 +88,7 @@ public Bundle getSupervisorPractitionerDetailsByKeycloakId(String keycloakUuid)
         return bundle;
     }
 
-    private Bundle getAttributedPractitionerDetailsByPractitioner(Practitioner practitioner) {
```

Comment:
We can change this to `protected` and it should still work, it has less visibility. 

Also add the annotation @VisibleForTesting